roles/aodh/templates/etc/serverspec/aodh_spec.rb

```diff
@@ -18,11 +18,6 @@
   it { should be_owned_by 'aodh' }
 end
 
-describe file('/var/log/aodh/aodh-expirer.log') do
-  it { should be_mode 644 }
-  it { should be_owned_by 'aodh' }
-end
-
```

Comment:
We're also no longer running aodh-expirer as a service, and instead as a periodic cron job.